packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -753,6 +768,23 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 		codeDiffMessage.replacing = false;
 	}
 
+	async function withoutFailedMessages(retry: () => Promise<void>) {
```

Comment:
Note that I ended up only using this function in `sendMessage`, as `initSupportChat` clears the chat anyway and the other methods don't interact with the chat directly (i.e. they rather send a new (quick) message) as far as I can tell. I could be totally wrong though, please let me know.

Reply:
Oh right, the ID is per message, not context, that's so much better 👏